src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -27,10 +27,8 @@ import ProviderMarker from '../components/markers/ProviderMarker';
 import { facilityTypes } from '../config';
 import { LocationType, FacilityType, BOUNDING_RADIUS } from '../constants';
 import { areGeocodeEqual /* areBoundsEqual */ } from '../utils/helpers';
-import {
-  isProduction,
-  facilityLocatorShowCommunityCares,
-} from '../utils/selectors';
+import { facilityLocatorShowCommunityCares } from '../utils/selectors';
+import { isProduction } from 'platform/site-wide/feature-toggles/selectors';
```

Comment:
not really necessary but we have an `isProduction` selector. The environment state is on the redux store now so we can use selectors for this if we want.